nixos/modules/system/boot/networkd.nix

```diff
@@ -303,6 +303,7 @@ let
   checkDhcpV6 = checkUnitConfig "DHCPv6" [
     (assertOnlyFields [
       "UseDns" "UseNTP" "RapidCommit" "ForceDHCPv6PDOtherInformation"
+      "PrefixDelegationHint"
```

Comment:
Withs->With in the commit message. Also, depending on timing, we should not merge this commit before the actual bump to systemd 244 or 245 has happened. Otherwise, users might be confused about using a option that's exposed in the NixOS module system, but silently ignored by systemd.

I'd propose fixing the other parts of this PR and dropping this commit, to not block it on the version bumps.